modules/menus/menu_settings/src/settings_menu.cpp

```diff
@@ -18,6 +18,7 @@ KEEP_FUNC SettingsMenu::SettingsMenu(Cursor& cursor)
           {"cursor color:", CURSOR_COLOR_INDEX, "Change cursor color", false, nullptr, MAX_CURSOR_COLOR_OPTIONS},
           {"font:", FONT_INDEX, "Change font", false, nullptr, FONT_OPTIONS_COUNT},
           {"drop shadows", DROP_SHADOWS_INDEX, "Adds shadows to all font letters", true, &g_dropShadows},
+          {"angle conversion", ANGLE_INDEX, "convert angle from hex to decimal", true, &g_angle},
           {"equip priority", ITEM_EQUIP_PRIORITY_INDEX, "Adjust priorities on item equips in practice saves", false},
```

Comment:
I would say "angles in decimal" for the check box name and "display angles in decimal instead of hex" as the description